pkg/components/contour/component_test.go

```diff
@@ -129,3 +130,18 @@ func TestConversion(t *testing.T) {
 		})
 	}
 }
+
+func TestEnvoyHostPort(t *testing.T) {
```

Comment:
We could have an e2e tests to verify the ports on Node IP address. what do you think?

Reply:
I don't think so it is needed, since this unit test is robust enough. Since the config generated here will be deployed so if we check here is as good as doing it on the cluster.